### ISSUE TITLE
Process of Training

### ISSUE THREAD

**Issue by mady143** [0dust/ResumeFilter#4]
Hi @0dust ,
I had clone your repository and tried to execute the problem i didn't find any issues or dependency related to the code but i just made some changes in training data added 1000 sample resumes and did training and i changed to 150 epochs i got an accuracy of 0.5 and then i did the prediction of some sample resumes but i haven't the  related info like name,email,mobile no,skills ,experience like may i know the issue what i am facing..

Note: while training process its showing like Train on 51 samples, validate on 22 samples
may i know the reason and can you help me in this to get the related info like experience from thge resume ...

Thanks and Regards,
Manikantha Sekhar...
